pkg/components/imguploader/s3uploader.go

```diff
@@ -77,10 +80,5 @@ func (u *S3Uploader) Upload(imageDiskPath string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
-	if u.region == "us-east-1" {
-		return "https://" + u.bucket + ".s3.amazonaws.com/" + key, nil
-	} else {
-		return "https://" + u.bucket + ".s3-" + u.region + ".amazonaws.com/" + key, nil
-	}
```

Comment:
if `u.bucket` contains `.`, e.g. `foo.bar`, the return url **was** `https://foo.bar.s3-us-west-2.amazonaws.com/`, it also causing slack does not render the image attachment , because the SSL certs common name is `*.s3-us-west-2.amazonaws.com` , which does **not** match `foo.bar.s3-us-west-2.amazonaws.com`